components/Modal/ClaimModal.tsx

```diff
@@ -0,0 +1,77 @@
+import { useState, useEffect, MouseEvent } from 'react';
+import { useAuthContext, useModalContext } from '../Provider';
+import {
+  Background,
+  ModalBox,
+  Section,
+  CloseIconContainer,
+  Title,
+  Description,
+  Row,
+  Spacer,
+  HalfButton,
+} from './Modal.styled';
+import ProtonSDK from '../../services/proton';
+import { formatPrice } from '../../utils';
+import { ReactComponent as CloseIcon } from '../../public/close.svg';
+
+export const ClaimModal = (): JSX.Element => {
+  const {
+    currentUser,
+    currentUserBalance,
+    updateCurrentUserBalance,
+  } = useAuthContext();
+  const { closeModal } = useModalContext();
+  const [error, setError] = useState<string>('');
+
+  useEffect(() => {
+    if (error) setError('');
+  }, []);
+
+  const withdraw = async () => {
+    try {
+      const res = await ProtonSDK.withdraw({
+        actor: currentUser ? currentUser.actor : '',
+        amount: `${currentUserBalance}`,
+      });
+
+      if (!res.success) {
+        throw new Error('Unable to make withdrawal.');
+      }
+
+      closeModal();
+      await updateCurrentUserBalance(currentUser.actor);
+    } catch (err) {
+      setError(err.message);
+    }
+  };
+
+  const handleBackgroundClick = (e: MouseEvent) => {
+    if (e.target === e.currentTarget) {
+      closeModal();
+    }
+  };
+
+  return (
+    <Background onClick={handleBackgroundClick}>
+      <ModalBox>
```

Comment:
Let's make sure the 
![Uploading Screen Shot 2021-03-23 at 10.45.33 AM.png…]()
modal is in front of the logo/navbar avatar in mobile view.